### DISCUSSION TITLE
Why does `tokio::time::Instant` copy documentation from `std::time::SystemTime`?

### DISCUSSION THREAD

**Discussion by amikhalev** [tokio-rs/tokio#2844] [General] [answered]
It looks like `tokio::time::Instant` takes its documentation from `std::time::SystemTime` and mentions that it is a measurement of the system clock. This is confusing considering it wraps `std::time::Instant`

**Comment by Darksonn** [comment]
Huh, that appears to be a mistake.

**Comment by taiki-e** [comment, marked as answer]
This is fixed by https://github.com/tokio-rs/tokio/commit/ec99e61945ae46edcb2ee3f282f49f661b63dcd0.
